mlflow/store/dbmodels/models.py

```diff
@@ -188,6 +189,47 @@ def to_mlflow_entity(self):
         return Run(run_info=run_info, run_data=run_data)
 
 
+class SqlExperimentTag(Base):
+    """
+    DB model for :py:class:`mlflow.entities.RunTag`.
+    These are recorded in ``experiment_tags`` table.
+    """
+    __tablename__ = 'experiment_tags'
+
+    key = Column(String(250))
+    """
+    Tag key: `String` (limit 250 characters). *Primary Key* for ``tags`` table.
+    """
+    value = Column(String(250), nullable=True)
```

Comment:
The maximum tag length configured here is less than the maximum length in the R documentation you included in https://github.com/mlflow/mlflow/pull/1632. I realize that the `value` column semantics for `SqlTag` also violate the documentation. IMO, we should use a maximum size of at least 500 bytes.

Reply:
Updating experimentTag to be 500 - I think the runTag update should be in a separate PR.